datafusion/sqllogictest/bin/sqllogictests.rs

```diff
@@ -159,6 +159,7 @@ async fn run_test_file_with_postgres(test_file: TestFile) -> Result<()> {
         relative_path,
     } = test_file;
     info!("Running with Postgres runner: {}", path.display());
+    setup_scratch_dir(&relative_path)?;
```

Comment:
without this complete mode didn't clear the scratch dir resulting in confusing errors